x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/dimension_container.tsx

```diff
@@ -54,24 +56,42 @@ export function DimensionContainer({
           className="lnsDimensionContainer"
         >
           <EuiFlyoutHeader hasBorder className="lnsDimensionContainer__header">
-            <EuiTitle size="xs">
-              <EuiButtonEmpty
-                onClick={closeFlyout}
-                data-test-subj="lns-indexPattern-dimensionContainerTitle"
-                id="lnsDimensionContainerTitle"
-                iconType="sortLeft"
-                flush="left"
-              >
-                <strong>
-                  {i18n.translate('xpack.lens.configure.configurePanelTitle', {
-                    defaultMessage: '{groupLabel} configuration',
-                    values: {
-                      groupLabel,
-                    },
+            <EuiFlexGroup
+              gutterSize="none"
+              alignItems="center"
+              className="lnsDimensionContainer__headerLink"
+              onClick={closeFlyout}
+            >
+              <EuiFlexItem grow={false}>
+                <EuiButtonIcon
```

Comment:
This looks a little strange because the button has the little y translation on hover while the heading doesn't: Can we make it look like the old version (no translation)?
![Kapture 2020-12-01 at 15 26 55](https://user-images.githubusercontent.com/1508364/100753327-eb1ee880-33e9-11eb-8d4d-97a3da6dff92.gif)